app/src/main/res/xml/account_settings_preferences.xml

```diff
@@ -29,6 +29,12 @@
             android:layout="@layout/checkbox_left_preference"
             android:title="@string/preferences_sync_bookmarks" />
 
+        <CheckBoxPreference
+            android:defaultValue="true"
```

Comment:
I think we set the default visibility for both of these to false to prevent it from flashing and disappearing depending on the prefs.